sympy/stats/joint_rv_types.py

```diff
@@ -8,11 +8,17 @@
     JointDistributionHandmade, MarginalDistribution)
 from sympy.stats.rv import _value_check, random_symbols
 
-# __all__ = ['MultivariateNormal',
-# 'MultivariateLaplace',
```

Comment:
`MultivariateNormal` and `MultivariateLaplace` should also be included.
It looks like the classes already exist, they just need methods for creation of the joint random variables.

Reply:
Actually they are created using `Normal` and `Laplace` functions in `crv_types`. So, they cannot be included here.

Reply:
OK, it seems the intent was to have a common interface for creation of continuous and joint RVs for them.